services/translateService.js

```diff
@@ -79,19 +60,19 @@ const setTranslateWithFallbackEngines = ({
     for await (const engine of enginesFiltered) {
       try {
         // Validate that the language is supported by the engine to avoid unnecessary network requests
-        validateLanguageIsSupportedByEngine(from, engine);
-        validateLanguageIsSupportedByEngine(to, engine);
+        const fromLanguageCode = getLanguageCodeByEngine(from, engine);
+        const toLanguageCode = getLanguageCodeByEngine(to, engine);
```

Comment:
Getting language code depending of the engine that is being used, so "no" could be "no" for google but "nb" for libreTranslate and Bing